hoot-core/src/main/cpp/hoot/core/io/ApiDb.cpp

```diff
@@ -384,4 +401,14 @@ QSqlQuery ApiDb::_execNoPrepare(const QString& sql) const
   return q;
 }
 
+long ApiDb::round(double x)
+{
+  return (long)(x + 0.5);
+}
+
+long ApiDb::round(double x, int precision)
+{
+  return (long)(floor(x * (10 * (precision - 1)) + 0.5) / (10 * (precision - 1)));
```

Comment:
I'll take a closer look at it.  That was old, pre-existing code that I just refactored into ApiDb....just cut and paste and didn't really look at it too closely :-)